packages/twenty-front/src/modules/object-record/record-field/meta-types/display/components/TextFieldDisplay.tsx

```diff
@@ -3,11 +3,17 @@ import { isFieldText } from '@/object-record/record-field/types/guards/isFieldTe
 import { TextDisplay } from '@/ui/field/display/components/TextDisplay';
 
 export const TextFieldDisplay = () => {
-  const { fieldValue, fieldDefinition } = useTextFieldDisplay();
+  const { fieldValue, fieldDefinition, wrap } = useTextFieldDisplay();
 
   const displayedMaxRows = isFieldText(fieldDefinition)
     ? fieldDefinition.metadata?.settings?.displayedMaxRows
     : 1;
 
-  return <TextDisplay text={fieldValue} displayedMaxRows={displayedMaxRows} />;
+  return (
+    <TextDisplay
+      text={fieldValue}
+      displayedMaxRows={displayedMaxRows}
+      wrap={wrap}
```

Comment:
I would use allowDisplayWrap (or wrap) here to compute displayedMaxRows directly, do we need two props?

Reply:
In a Table mode, if we have displayMaxRows = 5 then, unless wrap = false the table will still be mulitline. You see some other way to calculate that ?